cli/azd/pkg/infra/azure_resource_types.go

```diff
@@ -93,6 +94,8 @@ func GetResourceTypeDisplayName(resourceType AzureResourceType) string {
 		return "Cognitive Service"
 	case AzureResourceTypeSearchService:
 		return "Search service"
+	case AzureResourceTypeSpringApp:
+		return "Azure Spring Apps"
```

Comment:
For consistency with other service display names, consider 
```suggestion
		return "Azure Spring App"
```